Build failure: atuin

### Steps To Reproduce

Steps to reproduce the behavior:
1. build atuin on an x86-64 Mac

### Build log

https://gist.github.com/briankung/9956c2cfae998ca39d1c4f024e45dd55

### Additional context

The key part is probably this:

```
>   = note: Undefined symbols for architecture x86_64:
>             "_kCFURLVolumeAvailableCapacityForImportantUsageKey", referenced from:
>                 sysinfo::unix::apple::disk::get_list::h94c969b00aa897d2 in libsysinfo-5beb080e6b6ae604.rlib(sysinfo-5beb080e6b6ae604.sysinfo.26a22e64a9cf7170-cgu.10.rcgu.o)
>           ld: symbol(s) not found for architecture x86_64
>           clang-16: error: linker command failed with exit code 1 (use -v to see invocation)
>
>
> error: could not compile `atuin` (bin "atuin") due to 1 previous error
For full logs, run 'nix log /nix/store/hjvr6qnpdp5x1vqav760s7ggmfj4bs6l-atuin-18.3.0.drv'.
```

### Notify maintainers

- @SuperSandro2000
- @Sciencentistguy 
- @0x4A6F 

### Metadata

<!-- Please insert the output of running `nix-shell -p nix-info --run "nix-info -m"` below this line -->

 - system: `"x86_64-darwin"`
 - host os: `Darwin 24.0.0, macOS 10.16`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.24.10`
 - nixpkgs: `/nix/store/rxn26bq4xm2p0x4whkp94afhy1g88cm7-source`



---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/at/atuin/package.nix
+++ b/pkgs/by-name/at/atuin/package.nix
@@ -3,9 +3,8 @@
 , fetchFromGitHub
 , installShellFiles
 , rustPlatform
-, libiconv
 , buildPackages
-, darwin
+, apple-sdk_11
 , nixosTests
 }:
 
@@ -36,10 +35,7 @@ rustPlatform.buildRustPackage rec {
   nativeBuildInputs = [ installShellFiles ];
 
   buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [
-    libiconv
-    darwin.apple_sdk_11_0.frameworks.AppKit
-    darwin.apple_sdk_11_0.frameworks.Security
-    darwin.apple_sdk_11_0.frameworks.SystemConfiguration
+    apple-sdk_11
   ];
 
   preBuild = ''

